src/common/Containers.h

```diff
@@ -64,33 +64,63 @@ namespace acore
 
     namespace Containers
     {
-        template<class T>
-        void RandomResizeList(std::list<T>& list, uint32 size)
+        // replace with std::size in C++17
```

Comment:
we already use C++17 so this can be replaced now, no need to postpone for later

Reply:
https://github.com/azerothcore/azerothcore-wotlk/pull/5454

Reply:
@Winfidonarleyan where ?